docs/source/usage/tbg.rst

```diff
@@ -45,6 +45,25 @@ Feel free to copy & paste sections of the files below into your ``.cfg``, e.g. t
 .. literalinclude:: ../../TBG_macros.cfg
    :language: bash
 
+
+Automatic Grid Adjustment
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is important to note that PIConGPU has a feature to automatically adjust the grid to comply with PIConGPU's constraints.
+This feature is ACTIVE by default, meaning that unless explicit disabled PIConGPU does not guarantee that the grid you submitted is actually run.
+This can be deactivated by the commandline flag `--autoAdjustGrid=0`.
```

Comment:
should be `--autoAdjustGrid 0`

Reply:
Done

Reply:
PS: Both do exactly the same but if your suggestion is more in line with the style of the documentation I've changed it.